cli/internal/terraform/terraform/aws/.terraform.lock.hcl

```diff
@@ -0,0 +1,50 @@
+# This file is maintained automatically by "terraform init".
+# Manual edits may be lost in future updates.
+
+provider "registry.terraform.io/hashicorp/aws" {
+  version     = "4.36.1"
+  constraints = "~> 4.0"
```

Comment:
We should adjust the AWS version constraint to `"4.36.1"` instead of `"~> 4.0"`

Reply:
It does not make a difference when it comes to hash pinning but we can do this to prevent accidental upgrades :+1: .